pkg/autoscaler/autoscaler.go

```diff
@@ -434,4 +446,72 @@ var _ = Describe("[Feature:Machines] Autoscaler should", func() {
 			framework.WaitForMachineSetDelete(client, ms)
 		}
 	})
+
+	It("It scales from zero", func() {
+		defer GinkgoRecover()
```

Comment:
Don't think there's any go routines calling gomega matchers to need a recover? Or is this needed for calling `Expect` within and `Eventually`?

Reply:
I assumed this is needed for the `Eventually` async assertions

Reply:
Only needed to recover test failures within go routines as in the example below, don't think it's needed in this test suite, does it pass if you remove it?
```
go func() {
  defer GinkgoRecover()
  Expect(doSomething()).To(Succeed())
}()
```